File: TechRadar.Web/ClientApp/app/models/cycle.ts
export class Cycle {
    name: string;
    fullName: string;
    description: string;
    order: number;
    size: number;

    constructor(cycleInfo: any) {
        this.name = cycleInfo.name;
        this.fullName = cycleInfo.fullName;
        this.description = cycleInfo.description;
        this.order = cycleInfo.order;
        this.size = cycleInfo.size;
    }
}

File: TechRadar.Services/Models/Cycle.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace TechRadar.Services.Models
{
    [BsonIgnoreExtraElements]
    public class Cycle
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        [BsonElement("name")]
        public string Name { get; set; }

        [BsonElement("fullName")]
        public string FullName { get; set; }

        [BsonElement("description")]
        public string Description { get; set; }

        [BsonElement("order")]
        public int Order { get; set; }

        [BsonElement("size")]
        public int Size { get; set; }

    }
}
